<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KPF: KSE PathFinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="leKotor2"/>
        </item>
        <item>
         <widget class="QPushButton" name="bK2Browse">
          <property name="text">
           <string>Browse..</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="leKotor"/>
        </item>
        <item>
         <widget class="QPushButton" name="bK1Browse">
          <property name="text">
           <string>Browse..</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>SW KotOR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sw KotOR: TSL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="bExportINI">
          <property name="text">
           <string>Export to INI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bRescan">
          <property name="text">
           <string>Rescan Paths</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bUndoExport">
          <property name="text">
           <string>Undo Export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="menuOpen"/>
    <addaction name="menuDelete"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="menuAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="menuLogger"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menuExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="menuDelete">
   <property name="text">
    <string>&amp;Delete INI</string>
   </property>
  </action>
  <action name="menuOpen">
   <property name="text">
    <string>&amp;Open INI File</string>
   </property>
  </action>
  <action name="menuLogger">
   <property name="text">
    <string>Logger Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
